### PR TITLE
Crpl 1900 create google sheet connector

### PR DESCRIPTION
**Pull request by csabee1979**

### CODE REVIEW

File: sdk/google/spreadsheet/spreadsheet value updater/e54614aa-5347-49c3-9afc-e8202629f876.json [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+          "509ce9b8-1755-48e2-8a48-00aa5ad9c093": {
+            "22682fd2-ea2e-4bf3-b3f6-dea2cb95e29e": {
+              "73d008c4-b3e1-47f6-80d0-ebb4e53dd94c": {}
+            }
+          }
+        }
+      }
+    }
+  },
+  "description": "Writes data to a single range of  a given spreadsheet.\n\nExample:\n1. `session Id` receives \"spreadhseet_session\"@0 \n2. `spreadsheet Id` receives \"1_ewweewweFileID\"@0\n3. `update meta data` receives {\n \"range\": \"Sheet1!A1:B10\"\",\n \"valueInputOption\": \"RAW\" \n}@0\n4. `done` sends null@0 ",
```
Comment (lvass74) [inline sdk/google/spreadsheet/spreadsheet value updater/e54614aa-5347-49c3-9afc-e8202629f876.json:158]: Typo: "Íspreadhseet_session" should be "spread**sh**eet_session"

**Review by lvass74 — CHANGES_REQUESTED** [review]

File: sdk/google/spreadsheet/authenticated spreadsheet value updater/609d1deb-3db7-4638-b536-9aa58fc7d7d8.json [hunk 2]
```diff
@@ -0,0 +1,217 @@
+{
+  "name": "sdk/google/spreadsheet/Authenticated spreadsheet value updater",
+  "interface": {
+    "inputs": {
+      "23ea0957-d27a-456e-86a7-3e6391c12c6a": {
+        "type": "string",
+        "name": "session Id",
+        "description": "Receives the session id of the spreadsheet action.\n\nExample: \n\"spreadhseet_session\""
```
Comment (lvass74) [inline sdk/google/spreadsheet/authenticated spreadsheet value updater/609d1deb-3db7-4638-b536-9aa58fc7d7d8.json:8]: Typo in "spreadhseet_session"

File: sdk/google/spreadsheet/authenticator/f71cbfd4-054e-484f-a2b4-4ca8d15f4f58.json [hunk 3]
```diff
@@ -0,0 +1,156 @@
+{
+  "name": "sdk/google/spreadsheet/Authenticator",
+  "interface": {
+    "inputs": {
+      "5afcefbb-4dab-46b1-9823-deb800e402b5": {
+        "type": "string",
+        "name": "session Id",
+        "description": "Receives the session id of the spreadsheet action.\n\nExample: \n\"spreadhseet_session\""
```
Comment (lvass74) [inline sdk/google/spreadsheet/authenticator/f71cbfd4-054e-484f-a2b4-4ca8d15f4f58.json:8]: Typo in "spreadhseet_session"

File: sdk/google/spreadsheet/authenticator/f71cbfd4-054e-484f-a2b4-4ca8d15f4f58.json [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+            "954fd48c-afed-4a79-98a0-6fd7ca5c8ca2": {
+              "73d008c4-b3e1-47f6-80d0-ebb4e53dd94c": {}
+            }
+          }
+        }
+      }
+    }
+  },
+  "original": "b40028bc-c9de-41ea-a183-bfa0c413ebec",
+  "description": "Authenticates the service user to use google spreadsheet drive.\n\nExample:\n1. `session Id` receives \"spreadhseet_session\"@0 \n2. `email` receives  \"email@email.com\"@0\n3. `key` receives \"TopSecretKey\"@0\n4. `done` sends null@0 ",
```
Comment (lvass74) [inline sdk/google/spreadsheet/authenticator/f71cbfd4-054e-484f-a2b4-4ca8d15f4f58.json:145]: Typo in "spreadhseet_session"

**Review by lvass74 — CHANGES_REQUESTED** [review]

File: sdk/google/drive/authenticator/b40028bc-c9de-41ea-a183-bfa0c413ebec.json [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+            "eaadc9a9-28f4-48f0-b7d3-0c811b515f4a": "\"service_account_auth\"",
+            "type": "\"service_account_auth\""
+          }
+        },
+        "f1c53ade-f21f-412d-86a0-58c4a96900eb": {
+          "coordinates": [
+            -177,
+            -13
+          ],
+          "name": "forwarder inputs",
```
Comment (lvass74) [inline sdk/google/drive/authenticator/b40028bc-c9de-41ea-a183-bfa0c413ebec.json:74]: Should this rather be an action like "forward inputs"?

File: sdk/google/drive/file creator/a944f36b-bdfe-4e46-bfc2-a60548bbef56.json [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+      },
+      "67d952ea-512f-4028-8a2f-8f768bfa2e7a": {
+        "type": "{string:any}",
+        "name": "file meta data",
+        "description": "Receives the metadata of the file creation.\n\nExample in case of shared drive:\n\n{\n  \"name\": \"Test from CRANQ dynamic\",\n  \"driveId\": \"0AFITUOLHhs3TUk9PVA\",\n  \"corpora\": \"drive\",\n  \"parents\": [\n    \"FOLDER_ID\"\n  ],\n  \"mimeType\": \"application/vnd.google-apps.spreadsheet\"\n}\n\n\nExample in case of shared folder:\n\n{\n  \"name\": \"Test from CRANQ dynamic\",\n  \"parents\": [\n    \"FOLDER_ID\"\n  ]\n  \"mimeType\": \"application/vnd.google-apps.spreadsheet\"\n}\n"
+      },
+      "2890771f-bac0-4da3-998d-8f9530fcc98d": {
+        "type": "boolean",
+        "name": "supports all drives",
+        "description": "Receives `supportsAllDrives` parameter whether create should support both drives and shared drives.\n\nExample: \n\"true\""
```
Comment (lvass74) [inline sdk/google/drive/file creator/a944f36b-bdfe-4e46-bfc2-a60548bbef56.json:18]: Still unclear what is the `supportsAllDrives` we refer here. Is it a google SDK parameter?

File: sdk/google/drive/file creator/a944f36b-bdfe-4e46-bfc2-a60548bbef56.json [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+          "params": {
+            "ef31c1e9-cfd5-458b-8d3a-951e3db9c232": "[\"fileMetaData\",\"supportsAllDrives\"]"
+          }
+        },
+        "47e643d0-3326-489a-ab9d-b91d077f3a0a": {
+          "coordinates": [
+            -149,
+            27
+          ],
+          "name": "forwarder (triple)",
```
Comment (lvass74) [inline sdk/google/drive/file creator/a944f36b-bdfe-4e46-bfc2-a60548bbef56.json:74]: Should it be renamed to an action like "forward something"?

**Review by lvass74 — CHANGES_REQUESTED** [review]

File: sdk/google/drive/file creator/a944f36b-bdfe-4e46-bfc2-a60548bbef56.json (hunk 6 above)
Comment (csabee1979) [reply sdk/google/drive/file creator/a944f36b-bdfe-4e46-bfc2-a60548bbef56.json:18]: Yes. I don't want to put here all the google sdk documentations. The one who ones to use this basic node needs to be aware of the usage of the sdk

**Review by lvass74 — APPROVED** [review]

**Pull request merged**